app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,6 @@
 <resources>
     <string name="app_name">Picover</string>
+    <string name="home">Home</string>
```

Comment:
Just to inform: I've added issue to discuss the naming convention for the strings: https://github.com/intive/Picover/issues/13